x-pack/plugins/elastic_assistant/scripts/draw_graph_script.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ToolingLog } from '@kbn/tooling-log';
+import fs from 'fs/promises';
+import path from 'path';
+import {
+  ActionsClientChatOpenAI,
+  ActionsClientSimpleChatModel,
+} from '@kbn/langchain/server/language_models';
+import type { Logger } from '@kbn/logging';
+import { ChatPromptTemplate } from '@langchain/core/prompts';
+import { FakeLLM } from '@langchain/core/utils/testing';
+import { createOpenAIFunctionsAgent } from 'langchain/agents';
+import { getDefaultAssistantGraph } from '../server/lib/langchain/graphs/default_assistant_graph/graph';
+
+// Just defining some test variables to get the graph to compile..
+const testPrompt = ChatPromptTemplate.fromMessages([
+  ['system', 'You are a helpful assistant'],
+  ['placeholder', '{chat_history}'],
+  ['human', '{input}'],
+  ['placeholder', '{agent_scratchpad}'],
+]);
+const createLlmInstance = () => {
+  return new ActionsClientSimpleChatModel({ response: 'test' });
```

Comment:
This line here creates a type error but seems to be fine if its in scripts? I did not find any other way around it to make ActionClientSimpleChatModel work..
This was required because createLlmInstance is not exported, and I do no thave access to all the mocks from here either.